bindings/go/input/dir/method.go

```diff
@@ -97,6 +120,7 @@ func (i *InputMethod) ProcessSource(ctx context.Context, src *constructorruntime
 		return nil, fmt.Errorf("error converting resource input spec: %w", err)
 	}
 
+	// TODO: update call from https://github.com/open-component-model/open-component-model/pull/572
```

Comment:
while https://github.com/open-component-model/open-component-model/pull/572/files#diff-2d1251bfe35486a20c64d0f11feafb6da27931958411c17bf86215f222e0662fR85 works for files, it might not be applicable for the dir handling, since this happens in the input module